style.css

```diff
@@ -1 +1,428 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
 
+:root {
```

Comment:
Nice work using the root attribute to set up variables and make your code more organized! 😎